datacapturing/build.gradle

```diff
@@ -35,8 +35,6 @@ android {
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
 
-        versionCode 1 // this must be incremented if new apk should be pushed to play store
-        // this is shown to the users, see Confluence/Android Client for naming guideline
```

Comment:
Not used for libraries as this is only currently used when deployed to play store, i.e. only required in the app's build.gradle.